src/core/index.js

```diff
@@ -45,6 +45,8 @@ export {default as MapState} from './controllers/map-state';
 export {TRANSITION_EVENTS} from './lib/transition-manager';
 export {viewportLinearInterpolator, viewportFlyToInterpolator} from './lib/viewport-transition-utils';
 
+export {equals} from './math/equals';
```

Comment:
We need to consider what we want to export. Anything we add at this level will eventually become a public symbol when we move to "monorepo" setup. 

This doesn't feel like something deck.gl should export, and in this case the answer happens to be easy, now we can just `import {equals} from 'math.gl` and remove this function from deck.gl